django_mysql/models/functions.py

```diff
@@ -260,6 +260,31 @@ def __init__(self, expression, path=None, **extra):
         super(JSONLength, self).__init__(*exprs, output_field=output_field)
 
 
+class JSONSet(Func):
+    function = 'JSON_SET'
+
+    def __init__(self, expression, pairs):
+        from django_mysql.models.fields import JSONField
+
+        if isinstance(pairs, dict) and len(pairs.keys()) == 0:
+            raise ValueError('Pairs argument requires one key/value set defined')
+
+        exprs = [expression]
+
+        for path, value in pairs.iteritems():
```

Comment:
`iteritems` doesn't exist on Python 3, just use `items` since we expect the dict to be relatively small